plugins/faustwp/includes/replacement/functions.php

```diff
@@ -131,3 +131,106 @@ function is_wp_link_ajax_request(): bool {
 		&& ! empty( $_POST['action'] )
 		&& 'wp-link-ajax' === $_POST['action'] );
 }
+
+/**
+ * Get all site URLs for each possible HTTP protocol
+ */
+function faustwp_get_wp_site_urls() {
+
+	$site_url = site_url();
+	$host_url = wp_parse_url( $site_url, PHP_URL_HOST );
+
+	if ( ! is_string( $host_url ) ) {
+		return apply_filters( 'faustwp_get_wp_site_urls', array( $site_url ) );
```

Comment:
Maybe we can reduce the function length and get rid off one return?

Reply:
This probably isn't needed as the site url should be a string anyway.